Source/WebKit/UIProcess/RemoteLayerTree/RemoteScrollingTree.cpp

```diff
@@ -106,6 +106,22 @@ bool RemoteScrollingTree::scrollingTreeNodeRequestsKeyboardScroll(ScrollingNodeI
     return m_scrollingCoordinatorProxy->scrollingTreeNodeRequestsKeyboardScroll(nodeID, request);
 }
 
+void RemoteScrollingTree::scrollingTreeNodeWillStartScroll(ScrollingNodeID nodeID)
+{
+    RunLoop::main().dispatch([this, nodeID] {
```

Comment:
I think here in the base class we should take the iOS code, which assumes no scrolling thread. Override in `RemoteScrollingTreeMac` to do the hop to the main thread.

Reply:
done